integration-tests/tests/multichain/mod.rs

```diff
@@ -38,6 +38,15 @@ async fn test_triples_and_presignatures() -> anyhow::Result<()> {
             assert_eq!(state_0.participants.len(), 3);
             wait_for::has_at_least_triples(&ctx, 2).await?;
             wait_for::has_at_least_presignatures(&ctx, 2).await?;
+            // TODO: add test that checks #triples in datastore
```

Comment:
I would rather create a separate test. In that test, I would:
- Start 3 nodes
- Wait for a generation of N triples
- Stop all 3 nodes
- Start one of them and check that it has N triples (it will not be able to generate new triples, since other nodes are down)

Reply:
Yeah I wanted to do this but was not able to access the datastore correctly. The error is as shown a few lines down